ts/internal/version.ts

```diff
@@ -0,0 +1,4 @@
+// This is in its own file because the `with { type: "json" }` syntax breaks code formatting.
+import version from "./version.json" with { type: "json" };
```

Comment:
Should we consider just reading this with `Deno.readFileSync` instead if it causes issues with the formatter?

Reply:
That won't work in the case when the ts lib is imported from `https://garn.io/ts/...`. Also, it'd be hard to figure out the path when reading this from the cabal data files. I think it's easier to rely on deno's importing mechanism here.

We *do* have to make sure that the `version.json` file gets

- deployed with `just release-ts` (I checked that manually),
- included in the cabal data files (see the new change in `package.yaml`), and
- served for local development with `just fileserver` (also tested manually).

Reply:
Ah of course, forgot about the URL imports :confounded: